### PR TITLE
Support config set gitops

### PR DESCRIPTION
**Pull request by sanatnayar**
commands supported:
`qliksense config set gitops.enabled=<bool>`
`qliksense config set gitops.schedule=<string>`
`qliksense config set gitops.watchbranch=<string>`
`qliksense config set gitops.image=<string>`

### CODE REVIEW

**Comment by sanatnayar** [comment]
will work on refactoring setting configs with snother issue. Therefore this implementation will perhaps change in the future.

File: pkg/qliksense/context_configs.go [hunk 1]
```diff
@@ -225,8 +225,40 @@ func (q *Qliksense) SetOtherConfigs(args []string) error {
 			}
 			qliksenseCR.Spec.RotateKeys = rotateKeys
 			api.LogDebugMessage("Current rotateKeys after modification: %s ", qliksenseCR.Spec.RotateKeys)
+		case "gitops.enabled":
+			if qliksenseCR.Spec.GitOps == nil {
+				qliksenseCR.Spec.GitOps = &config.GitOps{}
+			}
+			if strings.EqualFold(argsString[1], "false") {
+				qliksenseCR.Spec.GitOps.Enabled = false
```
Comment (ffoysal) [inline pkg/qliksense/context_configs.go]: change its type to string and take value "yes"/"no"
also modify this https://github.com/qlik-oss/k-apis/blob/a9288454f9365a12523819765a1c4e5ac043a146/pkg/config/type.go#L79

**Review by ashwathishiva — APPROVED** [review]

File: pkg/qliksense/context_configs.go [hunk 2]
```diff
@@ -225,8 +225,32 @@ func (q *Qliksense) SetOtherConfigs(args []string) error {
 			}
 			qliksenseCR.Spec.RotateKeys = rotateKeys
 			api.LogDebugMessage("Current rotateKeys after modification: %s ", qliksenseCR.Spec.RotateKeys)
+		case "gitops.enabled":
+			if qliksenseCR.Spec.GitOps == nil {
+				qliksenseCR.Spec.GitOps = &config.GitOps{}
+			}
+			qliksenseCR.Spec.GitOps.Schedule = argsString[1]
```
Comment (ffoysal) [inline pkg/qliksense/context_configs.go]: can we validate the value if it is "yes" or "no" only ??

File: pkg/qliksense/context_configs.go [hunk 3]
```diff
@@ -225,8 +225,32 @@ func (q *Qliksense) SetOtherConfigs(args []string) error {
...
+			api.LogDebugMessage("Current gitOps enabled status : %s ", qliksenseCR.Spec.GitOps.Enabled)
+		case "gitops.schedule":
+			if qliksenseCR.Spec.GitOps == nil {
+				qliksenseCR.Spec.GitOps = &config.GitOps{}
+			}
+			qliksenseCR.Spec.GitOps.Schedule = argsString[1]
```
Comment (ffoysal) [inline pkg/qliksense/context_configs.go:249]: we can validate if the cron schedule is correct or not ??
https://github.com/robfig/cron/blob/208b4f76bfdd237a814619382b19c85097fa9142/cron.go#L30

Comment (sanatnayar) [reply pkg/qliksense/context_configs.go:249]: cron.ParseStandard() should validate based on standard cron scheduling syntax

**Review by ffoysal — APPROVED** [review]

**Pull request merged**